diff --git a/node_modules/gatsby/dist/utils/eslint-config.js b/node_modules/gatsby/dist/utils/eslint-config.js
index 07797c7..b15c194 100644
--- a/node_modules/gatsby/dist/utils/eslint-config.js
+++ b/node_modules/gatsby/dist/utils/eslint-config.js
@@ -16,8 +16,9 @@ const eslintRequirePreset = require.resolve(`./eslint/required`);
 
 const eslintRequiredConfig = {
   rulePaths: [eslintRulePaths],
-  useEslintrc: false,
+  useEslintrc: true,
   baseConfig: {
+    parser: require.resolve(`babel-eslint`),
     parserOptions: {
       ecmaVersion: 2018,
       sourceType: `module`,
